api/spec/controllers/mno_enterprise/jpi/v1/subscriptions_controller_spec.rb

```diff
@@ -1,12 +1,17 @@
 require 'rails_helper'
 
 module MnoEnterprise
-  describe Jpi::V1::SubscriptionsController, type: :controller do
+  describe Jpi::V1::SubscriptionsController, type: :controller, ignore: true do
```

Comment:
I think we can ignore the relationship declaration for now.
I am trying to use it for read-only anyway.